### PR TITLE
A useful option

### PR DESCRIPTION
**Pull request by devcustrom**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -139,8 +139,15 @@ const getWorksheet = (jsonSheet: IJsonSheet, settings: ISettings): WorkSheet =>
...
   const filename = `${settings.fileName ?? "Spreadsheet"}.xlsx`
   const writeOptions = settings.writeOptions ?? {}
+  
+  if(settings.writeMode === 'write') {
+    return write(workbook, writeOptions)
+  } else if(settings.writeMode === 'writeFile') {
+    return writeFile(workbook, filename, writeOptions)
+  } else {
+    return writeOptions.type === "buffer" ? write(workbook, writeOptions) : writeFile(workbook, filename, writeOptions)
+  }
```
Comment (elyse0) [inline src/index.ts:150]: If you are returning you don't need `else` in each case

File: README.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ You can see a live example of it working on any of this sites (there are many ju
 ## Usage
 
 ```js
+import xlsx from "json-as-xlsx-wm"
```
Comment (elyse0) [inline README.md:14]: "json-as-xlsx-wm" is your package? 🤔

File: src/index.ts (hunk 1 above)
Comment (devcustrom) [reply src/index.ts:150]: I left `else` for backward compatibility

File: README.md (hunk 2 above)
Comment (devcustrom) [reply README.md:14]: Yes, this is my package. The usual inattention. Need to fix the package name

**Comment by devcustrom** [comment]
great, now you can abandon my fork in working projects

**Pull request merged**